client/settings/settings-manager/style.scss

```diff
@@ -0,0 +1,52 @@
+body {
+	/* this can be removed after all the settings sections are converted to the new layout */
+	&.woocommerce_page_wc-settings {
+		&.woocommerce_page_wc-settings-checkout {
+			background: #fff;
+			.settings-content {
+				background: #fff;
+			}
+			#wpbody-content {
+				background: #fff;
+			}
+			#wpcontent {
+				background: #fff;
+			}
+
+			.woopay-settings {
+				.woopay-settings__appearance-card-settings {
```

Comment:
Similar to above, please combine the selectors with comma to prevent duplication.

Reply:
changed in [ee1a5c6](https://github.com/Automattic/woocommerce-payments/pull/10578/commits/ee1a5c667ebf360eac3c476d55870eacc1c883e3) . thx